container/features/src/main/resources/features.xml

```diff
@@ -1135,9 +1139,14 @@
         <bundle>mvn:org.opennms.features.flows.classification/org.opennms.features.flows.classification.shell/${project.version}</bundle>
     </feature>
     <feature name="opennms-api-layer" version="${project.version}" description="OpenNMS :: Features :: API Layer">
+        <feature>opennms-core</feature>
         <feature>opennms-health</feature>
         <feature>opennms-situation-feedback-api</feature>
+        <feature>opennms-graph-api</feature>
+        <feature>opennms-topology-api</feature>
+        <feature>opennms-graph-provider-topology</feature>
         <feature version="${opennmsApiVersion}">opennms-integration-api</feature>
+        <feature>hibernate36</feature>
```

Comment:
This one stands out. Curious as to what triggered these updates.

Reply:
I am confused. I could not reproduce the hibernate dependency requirement. Even worse, I could not reproduce any of the requirements that for which I had added feature / bundle dependencies to resolve them!

I propose to let `features.xml` just like it was, i.e. no upgrade to v1.6.0.

Reply:
Works for me - the other changes look good.